src/main/java/module/transfer/XMLParser.java

```diff
@@ -60,7 +61,7 @@ public static List<PlayerTransfer> getAllPlayerTransfers(int playerId) {
 
                 // Get the player info
                 final Element playerElement = (Element) container.getElementsByTagName("Player").item(0); //$NON-NLS-1$
-                final int playerid = XMLManager.xmlIntValue(playerElement, "PlayerID"); //$NON-NLS-1$
+                playerId = XMLManager.xmlIntValue(playerElement, "PlayerID"); //$NON-NLS-1$
```

Comment:
I don't get the benefit from reusing the parameter of the function and overwrite the value.
Can you explain?
But yes, the name of variable should be camel case.

Reply:
Reason for the change was to adjust the spelling. I don't know if reusing the parameter has some disadvantages. If parameter and the return value of the xml parsing differs an error on HT side occured. They should always be identical.

Reply:
Okay. We should refactor this later when the parsing code gets improved - okay from my side.